assets/files.wxs

```diff
@@ -3065,6 +3065,23 @@
       <Component Id="cmpEC42F4F98D70437BB0E290A0E0FE1775" Guid="{5483092a-12c2-4e6c-b730-be491b8909c0}">
```

Comment:
@SteveL-MSFT, your last commit had 4 failures in `PowerShell-CI-windows`
WindowStyle argument.-WindowStyle Normal should work on Windows
```powershell
This command cannot be run due to the error: The system cannot find the file specified..
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 872
872:             $ps = Start-Process [System.AppDomain]::CurrentDomain.FriendlyName -ArgumentList "-WindowStyle $WindowStyle -noexit -interactive" -PassThru

```
WindowStyle argument.-WindowStyle Minimized should work on Windows
```powershell
This command cannot be run due to the error: The system cannot find the file specified..
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 872
872:             $ps = Start-Process [System.AppDomain]::CurrentDomain.FriendlyName -ArgumentList "-WindowStyle $WindowStyle -noexit -interactive" -PassThru

```
WindowStyle argument.-WindowStyle Maximized should work on Windows
```powershell
This command cannot be run due to the error: The system cannot find the file specified..
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 872
872:             $ps = Start-Process [System.AppDomain]::CurrentDomain.FriendlyName -ArgumentList "-WindowStyle $WindowStyle -noexit -interactive" -PassThru

```
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @{InputObject=System.Threading.Timer.dll; SideIndicator=<=}.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\Startup.Tests.ps1: line 126
126:         $diffs | Should -BeExactly $null
```